iroh/src/magicsock/relay_actor.rs

```diff
@@ -862,6 +903,17 @@ impl RelayActor {
                     debug!("shutting down");
                     break;
                 }
+                Some(message) = self.relay_disco_recv_rx.recv() => {
+                    self.msock.
+                        handle_disco_message(
```

Comment:
I'm a bit worried doing this *in* the RelayActor.  This actor needs to be able to move packets **fast**.  I would like it to avoid doing any processing like decrypting messages and taking a whole bunch of locks (what this call does).

Reply:
Agreed. A separate task that's spawned from the magicsock maybe that does just that?